hw/ip/keymgr/data/keymgr.hjson

```diff
@@ -304,58 +298,79 @@
       ]
     },
 
-    { name: "ROM_EXT_DESC_EN",
-      desc: "Register write enable for ROM_EXT_DESC",
+    // Per https://docs.opentitan.org/doc/security/specs/identities_and_root_keys/#creator-root-key
```

Comment:
nit: if this is not needed, perhaps remove or add a TODO to clean up later

Reply:
yep! i just sync'd up with @alphan today and i think we're good with removing this.
I will do so.